src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -32,4 +36,15 @@ public List<BookDto> getAll() {
     public BookDto getBookById(@PathVariable Long id) {
         return bookService.getBookById(id);
     }
+
```

Comment:
you can change RequestMapping(value = "/api/books")
it is a little better to delete /api from here and add to your application.properties this prop:
server.servlet.context-path=/api

Reply:
right, agree